src/main/java/frc/robot/subsystems/base/common/motors/FalconBuilder.java

```diff
@@ -0,0 +1,221 @@
+package frc.robot.subsystems.base.common.motors;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import frc.robot.utilities.FalconConfiguration;
+
+import java.util.Arrays;
+
+/**
+ * Provide a builder for Falcon motors.
+ *
+ * @author Barel
+ * @version 1.0
+ * @since 2020-off
+ */
+public class FalconBuilder {
+    private final TalonFX motor;
+
+    /**
+     * Create a builder for the falcons.
+     *
+     * @param port         the port of the falcon.
+     * @param inverted     whether the falcon is inverted.
+     * @param openLoopRamp Configures the open-loop ramp rate.
+     */
+    public FalconBuilder(int port, boolean inverted, double openLoopRamp) {
```

Comment:
Haven't you made already a falcon config method?

Reply:
Where?

Reply:
I added the Falcon configuration class because I wanted to compile the code, and it was impossible at the time. I will remove it ASAP.

Reply:
Make sure next time you don't commit it to the repository.